dashboard/test/ui/features/script_overview.feature

```diff
@@ -29,3 +29,22 @@ Feature: Script overview page
 
     # Make sure we only see student progress, not teacher progress.
     Then I verify progress for stage 29 level 4 is "not_tried"
+
+  Scenario: Script overview contents
+    Given I create a student named "Jean"
+    And I am on "http://studio.code.org/s/allthethings"
+    # make sure we are in summary view and the page has finished loading
+    And I wait until element "td:contains(Maze)" is visible
+    # verify name format in summary view
+    And element "td:contains(2. Maze)" is visible
```

Comment:
I wrote these UI tests when I thought I might try to refactor Script#localized_title and localized_name, but decided to keep them in here anyway even though I am no longer touching that codepath.